cartoframes/cartoframes.py

```diff
@@ -8,5 +8,5 @@ def __init__(self, base_url=None, api_key=None, creds=None, session=None,
         self.cc = CartoContext(base_url=base_url, api_key=api_key, creds=creds, session=session, verbose=verbose)
```

Comment:
I know this isn't in your PR, but as long as we're doing refactoring, how about changing the attribute name to `self.context` instead of `self.cc`? Besides, you change the name of the class to `Context` :)

Reply:
Yes, I had the same conversation with @oleurud. Agree :)

Note: I would keep the previous `cartoframes.CartoContext` to keep the previous API.

Reply:
I will do it in a different PR, this one is really big

Reply:
Agree with @oleurud. We should try to close this one, and open a ticket for every new ticket or discussion about new changes.